Admin/StaticContentAdmin.php

```diff
@@ -49,18 +46,13 @@ protected function configureFormFields(FormMapper $formMapper)
     {
         $formMapper
             ->with('form.group_general')
-                ->add('parentDocument', TreeModelType::class, array(
-                    'root_node' => $this->getRootPath(),
-                    'choice_list' => array(),
-                    'select_root_node' => true,
-                ))
-                ->add('name', TextType::class)
-                ->add('title', TextType::class)
-                ->add(
-                    'body',
-                    $this->ivoryCkeditor ? CKEditorType::class : TextareaType::class,
-                    $this->ivoryCkeditor
-                )
+                ->add('static_content', StaticContentType::class, [
+                    'inherit_data' => true,
```

Comment:
let's configure this one in the StaticContentType, I can't imagine a case where we use StaticContentType without wanting it to inherit_data.